test/fixtures/output/connect/custom-plugin/version/create-invalid-is-beta.golden

```diff
@@ -0,0 +1 @@
+Error: only is-beta values "true" or "false" are allowed
```

Comment:
Is this file still needed? I don't think this error is printed anymore.

Reply:
you are right, remove this file.